### PR TITLE
Try to patch comment

### PR DESCRIPTION
**Pull request by kitagry**
I fixed patch option. Please review.

### CODE REVIEW

**Comment by kitagry** [comment]
Sorry, I am writing some tests. And then, I'll request a review.

File: pkg/terraform/template.go [hunk 1]
```diff
@@ -187,80 +241,25 @@ func generateOutput(kind, template string, data map[string]interface{}, useRawOu
 
 // Execute binds the execution result of terraform command into template
 func (t *Template) Execute() (string, error) {
-	data := map[string]interface{}{
```
Comment (kitagry) [inline pkg/terraform/template.go:190]: Isn't this map necessary?

**Comment by kitagry** [comment]
@hirosassa Please review!

**Comment by hirosassa** [comment]
LGTM! Thanks!

**Pull request merged**